content/web-frameworks/gatsby/why-gatsby.md

```diff
@@ -31,6 +35,17 @@ showToc: true
   - [GitHub Action workflow for GitHub Pages](/programming-cookbook/github-actions/deploy-to-github-pages)
   - [GitHub Action workflow for CloudFlare Workers](/programming-cookbook/github-actions/deploy-to-cloudflare-workers)
 
+# Gatsby don't be suitable if:
+
+- The web admin is not a developer. To modify the content or design, it requires programming skill.
+- You want admin control panel to modify the content.
+- Your content will be changed quite often, it'll make difficult to modify the content when compare to use admin panel.
```

Comment:
You use the word _admin control panel_ from previous bullet. It better to use consistent word. For example:

```
..., it's more difficult to modify content than using admin control panel.
```

Reply:
Oh, thanks a lot. Your eye is sharpen.  😊